### DISCUSSION TITLE
Floating block removal event.

### DISCUSSION THREAD

**Discussion by Sonicscream** [piqueserver/piqueserver#685] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
There isn't a documented way to keep track of blocks that get destroyed when they lose a path to ground (blocks that end up floating which get removed).

**Describe the solution you'd like**
I propose another "event", similar to:

https://github.com/piqueserver/piqueserver/blob/2396af5b6f43331d1144c015fb3fc9164900ef59/piqueserver/player.py#L202
and
https://github.com/piqueserver/piqueserver/blob/2396af5b6f43331d1144c015fb3fc9164900ef59/piqueserver/player.py#L229

but instead the parameters is/are a list of blocks that became floating and then destroyed.

**Comment by Sonicscream** [comment]
One particular example of why I find this would be useful is for managing griefing in a server where there are tall structures that are easy to topple by digging out their bottom blocks.

I checked scripts/blockinfo.py and I do not see where it counts fallen blocks towards a players destroyed block count.

**Comment by NotAFile** [comment]
I don't think they should be counted against the players block count directly, btw. That makes the metric pretty useless.

**Comment by Sonicscream** [comment]
Yeah, you could have a separate metric just for the fallen blocks. I think it would give a sense of how much destruction they are actually causing.
In my opinion it is difficult to tell if a person destroyed only a few blocks or if they also collapsed a lot of blocks too.
